src/person/serializers.py

```diff
@@ -45,32 +47,47 @@ class UserSerializer(serializers.ModelSerializer):
     interests = InterestSerializer(source="person.interests", many=True)
     groups = GroupSerializer(source="person.groups", many=True)
     has_onboarded = serializers.BooleanField(source="person.has_onboarded")
-    matches = serializers.SerializerMethodField("get_match_history")
+    current_match = serializers.SerializerMethodField("get_current_match")
+    previous_match = serializers.SerializerMethodField("get_previous_match")
 
     def get_goals(self, user):
         if user.person.goals is None:
-            return None
-        print(user.person.goals)
+            return []
         goals = json.loads(user.person.goals.replace("'", '"'))
-        return goals or None
+        return goals
+
+    def get_has_onboarded(self, user):
+        if user.person.has_onboarded is None:
```

Comment:
you can just return `user.person.has_onboarded` because you've guaranteed that `has_onboarded` has a `False` default value and you explicitly set it to `False` for every user you create that's going through onboarding.